### ISSUE TITLE
Attempting to send commands throw exceptions when a session hasn't started

### ISSUE THREAD

**Issue by sidoh** [happyleavesaoc/python-limitlessled#15]
I noticed this with HomeAssistant. When my hub device was unplugged, the entire lights section (I think including some non-milight devices) was failing to initialize with an error that looked like this:

```
Traceback (most recent call last):
  File "/usr/src/app/homeassistant/helpers/entity_component.py", line 153, in _async_setup_platform
    entity_platform.schedule_add_entities, discovery_info
  File "uvloop/future.pyx", line 230, in __iter__ (uvloop/loop.c:110600)
  File "uvloop/future.pyx", line 432, in uvloop.loop.BaseTask._fast_wakeup (uvloop/loop.c:113980)
  File "uvloop/future.pyx", line 101, in uvloop.loop.BaseFuture._result_impl (uvloop/loop.c:108900)
  File "/usr/local/lib/python3.5/concurrent/futures/thread.py", line 55, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/app/homeassistant/components/light/limitlessled.py", line 118, in setup_platform
    lights.append(LimitlessLEDGroup.factory(group))
  File "/usr/src/app/homeassistant/components/light/limitlessled.py", line 178, in factory
    return LimitlessLEDRGBWWGroup(group)
  File "/usr/src/app/homeassistant/components/light/limitlessled.py", line 314, in __init__
    self.group.on = True
  File "/usr/local/lib/python3.5/site-packages/limitlessled/group/__init__.py", line 83, in on
    cmd = self.command_set.off()
  File "/usr/local/lib/python3.5/site-packages/limitlessled/group/commands/v6.py", line 303, in off
    return self._build_command(0x04, 0x02)
  File "/usr/local/lib/python3.5/site-packages/limitlessled/group/commands/v6.py", line 111, in _build_command
    self._group_number)
  File "/usr/local/lib/python3.5/site-packages/limitlessled/group/commands/__init__.py", line 44, in __init__
    self._bytes = bytearray(bytes)
TypeError: an integer is required
```

It looks like this is because it failed to fetch session ID bytes (`wb1` and `wb2` in the docs/code), so they remain set to `None`. The code that builds a command packet tries to build a byte array with the session ID values being set to `None`, which causes the above failure.

Seems like it might make sense to skip or queue commands that are being sent before a session has successfully started? If it makes sense to fail, seems like it'd be best to do so with a more helpful error message.

**Comment by happyleavesaoc** [comment]
@soldag

**Comment by soldag** [comment]
I decided for queuing commands, if session is not initialized yet, or connection is lost.

Fixed in ba0b18a25fe2d332711d8764caa0fbbb829f898e.

**Comment by sidoh** [comment]
Awesome. Thanks for fixing!

Does it hold onto queued commands indefinitely? I could imagine this being a little unexpected if, for example:

1. a hub goes offline.
2. user spams a a bunch of commands wondering why they're not working.
3. reboots hub, lights everywhere go nuts.

I'm not exactly sure this is what would happen in the usual case where HomeAssistant (or whatever) already has a session open, but the hub goes offline. Maybe sending the command would just throw an exception, which would make sense.
